src/java/fr/paris/lutece/plugins/blog/web/BlogJspBean.java

```diff
@@ -571,7 +571,7 @@ public String doRemoveTag( HttpServletRequest request )
                 return JsonUtil.buildJsonResponse( new JsonResponse( RESPONSE_BLOG_LOCKED ) );
             }
 
-        if ( RBACService.isAuthorized( Tag.PROPERTY_RESOURCE_TYPE, strIdTag, Tag.PERMISSION_DELETE, getUser( ) ) )
+        if ( RBACService.isAuthorized( Tag.PROPERTY_RESOURCE_TYPE, strIdTag, Tag.PERMISSION_DELETE, getUser( ) == null ? AdminUserService.getAdminUser( request ) : getUser() ) )
```

Comment:
What is this fix for?

Reply:
Can you please create a jira for describing the functionality you want to add and do the PR with the jira number?